pkg/api/rest/types.go

```diff
@@ -60,7 +81,7 @@ type podStrategy struct {
 
 // Pods is the default logic that applies when creating and updating Pod
 // objects.
-var Pods RESTCreateStrategy = podStrategy{api.Scheme, api.SimpleNameGenerator}
+var Pods = podStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
Is Strategy still the right name for this? It's an ObjectTyper and NameGenerator? Why are these separate, anyhow?

Could we please adopt a convention for declaring what interfaces receivers implement? #2992